src/main/java/core/basesyntax/service/impl/DataConverter.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface DataConverter {
```

Comment:
The interface name `DataConverter` is a bit generic. Consider renaming it to something more specific to its purpose, such as `TransactionConverter`, to clarify what kind of data it is intended to convert.